README.md

```diff
@@ -1423,6 +1432,16 @@ For multisegment variable lengths tests:
 ![](performance/images/exp3_multiseg_wide_time.svg) ![](performance/images/exp3_multiseg_wide_efficiency.svg)    
 ![](performance/images/exp3_multiseg_wide_records_throughput.svg) ![](performance/images/exp3_multiseg_wide_mb_throughput.svg)
 
+
+### How to generate Code coverage report
+```sbt
+sbt jacoco
+```
+Code coverage will be generated on path:
+```
+{local-path}\fixed-width\target\scala-2.XY\jacoco\report\html
+```
+
```

Comment:
This might be extra

Reply:
Yes, I was in doubt if it even needed to place in readme when I saw that file. Remove it?